### DISCUSSION TITLE
[1089090] 海贼无双4[本体+全DLC]

### DISCUSSION THREAD

**Discussion by BTSEAM** [pjy612/SteamManifestCache#293] [Share]
本体
`appid：1089090`
DLC
```
appid：1130841
appid：1203040
appid：1203041
appid：1203042
appid：2449140
appid：2449250
appid：2452120
```

key
```
1089091;49964ca34e77b3aee8889132d8de8da22521d34a3cdfbfda9c74bdfb995a6bd3
1130840;139786567849d65b72a9e95a65a33c440aab4f87e84f16c5967eece9f002c31a
1203040;23721734da64a0e0eed9e25288becfacbd86526db03272187e504b5a454ce809
1203041;bf69c2c5c04fd1c592fc1d9ff6100309b370feb381996dfc7586d5a0fbc5cc3e
1203042;daa2e5d518b63ac6b8878c5ea4c540165ffdc4f7dfa078bfc99e165da8803a65
2449140;a5b90a998cb9fd64fa11da5687004b175f17d545d9b5b2d68e8631c6bc2a841d
2449150;cb4ad443ede1955f0390dfd14006088a10213c6a97269acc9454b933b2395d30
2452120;b54bcf33357770b5d13a1e145d50d48aaa4dfed4be53b630f92cb1e4864838be
```

[1089090_config.vdf.txt.txt](https://github.com/pjy612/SteamManifestCache/files/13887930/1089090_config.vdf.txt.txt)
[清单.zip](https://github.com/pjy612/SteamManifestCache/files/13885265/default.zip)

code:425170FE57C049B6
E-mail:1816273998@qq.com

**Comment by pjy612** [comment]
请检查密钥
https://steamdb.info/app/1089090/depots/

**Reply by BTSEAM** [reply]
是缺密钥吗？

**Reply by BTSEAM** [reply]
现在呢

**Reply by pjy612** [reply]
一样
本体的 1089091 没有的话 能玩？

**Reply by BTSEAM** [reply]
只找到了清单中有9091，key没找到

**Reply by huihui03** [reply]
我看了下你的附件，为什么要把1089091自己换成1089090呢？![Screenshot_2024-01-10-07-13-53-11_5a36f6e0f8b96c8fbbd9efc54bb8de1e.jpg](https://github.com/pjy612/SteamManifestCache/assets/40056970/744c4043-ce50-4876-b492-4bfcae55199d)

**Reply by huihui03** [reply]
按照附件的config重新编辑keys就成了

**Reply by pjy612** [reply]
@huihui03 最开始的时候松散一些。
现在 忙起来了 看得少，就需要他们对着格式弄了。

**Comment by pjy612** [comment]
所以 
```
1506601;e73d6c085fd485de54e5f2dfae869f55445e570b953297de711aff897c3d6070
1527740;60f140de37544926389e2dcafa9566ff657fc7a4ca8b478e379f19c465021a95
```
这俩你从哪儿抄来的？和海贼有关吗？ @BTSEAM

**Reply by BTSEAM** [reply]
![image](https://github.com/pjy612/SteamManifestCache/assets/135128490/624f1f97-2aa9-4661-b6da-8e81e6317424)
里面1203042没了，这俩加到里面了
